FirebaseAuthUI/FUIAuth.m

```diff
@@ -309,12 +311,24 @@ - (void)signInWithEmailHint:(NSString *)emailHint
                               nil];
     if ([supportedProviders containsObject:existingFederatedProviderID]) {
       if ([existingFederatedProviderID isEqualToString:FIREmailAuthProviderID]) {
-        FUIAuth *authUI = [[FUIAuth alloc]initWithAuth:tempAuth];
-        // Email password sign-in
-        FUIPasswordSignInViewController *controller =
-            [[FUIPasswordSignInViewController alloc] initWithAuthUI:authUI email:emailHint];
-        controller.onDismissCallback = completion;
-        [presentingViewController pushViewController:controller];
+
+        [FUIAuthBaseViewController showSignInAlertWithEmail:emailHint
+                                          providerShortName:@"Email/Password"
```

Comment:
These strings will eventually have to be localized (can be in a follow up PR).

Reply:
Will do, had added as part of the PR description :)